### DISCUSSION TITLE
Compatibility Issue with pg_dump on Supabase

### DISCUSSION THREAD

**Discussion by LazarManyc** [supabase/supabase#22575] [Questions] [answered]
Hello,

I hope you're doing well. I'm currently facing an issue with using pg_dump to export the schema of my PostgreSQL database on Supabase. When attempting to use pg_dump, I receive the following error message:

```
pg_dump: error: server version: 15.1 (Ubuntu 15.1-1.pgdg20.04+1); pg_dump version: 14.11 (Ubuntu 14.11-0ubuntu0.22.04.1)
pg_dump: error: aborting because of server version mismatch
```

It seems that the version of pg_dump I'm using is not compatible with the version of the PostgreSQL server running on Supabase.

I've read that using Docker might be a solution, but I'm not familiar with Docker. Is there another solution to export the schema of my database using a compatible version of pg_dump?

I'd appreciate any guidance or suggestions on how to resolve this issue.

Thank you for your help.

**Comment by GaryAustin1** [comment, marked as answer]
You need to install the same version of Postgres locally for pg_dump to work.
https://docs.digitalocean.com/support/how-do-i-fix-this-pg_dump-error-pg_dump-error-aborting-because-of-server-version-mismatch
